content/en/users/online-storage/object-storage/_index.md

```diff
@@ -140,34 +244,29 @@ $ fedcloud openstack --site IFCA-LCG2 --vo vo.access.egi.eu ec2 credentials crea
 +------------+------------------------------------------------------------------------------------------------------------------------------------------+
 ```
 
-<!-- markdownlint-enable line-length -->
```

Comment:
The line length exception is mainly just for tables, not for code.
Please keep it here, and for the shell excerpt use line returns + backslash to split over multiple lines.

Reply:
I disagree. shell commands are not code. if I break them using backslashes just for readability, that means those lines will stay broken, irrespective of how the browser is resized.

Reply:
what I could do is wrap each shell example in individual disable/enable html comments (to keep the rule active for regular text. 

although, in my opinion there are some linter rules that are enforced now but shouldn't (the line length and no space at line end are two of them). the contributions should be made as easy as possible, and only enforce those rules that are important (e.g. no broken links - but even for this we don't have the right tools: the link checker is dumb and does not understand how Hugo renders content, which forces us to use sections instead of pages for everything, and even fails to validate perfectly valid links in at least one case, where I had to disable link checking).

Reply:
Long lines in shell excerpt are not really a good/usual practice and not everyone is using a big screen, "broken" lines is the standard way to render them.

People are also editing the source of files in terminals or small editors and unlimited lines are definitely not handy/clean.

We are also relying on prettier so that we are using quite standard and known style rule and don't have to discuss every rules, and people can usually just use prettier.

If you have another link checker to propose that work best we will be happy to switch to it, but until we have a better alternative we will stay on this one as it already helps more than having nothing at all.

Reply:
> Long lines in shell excerpt are not really a good/usual practice and not everyone is using a big screen, "broken" lines is the standard way to render them.

I agree. that is why the commands itself are as short as possible. only the command output in my examples is wide. breaking that would be wrong and misleading.

Reply:
> If you have another link checker to propose

unfortunately, I don't have a better one. but link checking is very important, and it makes sense to struggle with the tool we have. some of the other rules, however, are a different story.

Reply:
I gave up on prettier, it messes up the formatting in some cases. but more importantly, it makes reading the docs in a text editor a pain (after it applies rigorous follow-the-rules formatting). so to be able to organize my thoughts, I prefer editing the source files manually, and keep text human readable so I can easily scan then and write/correct/improve it.

Reply:
Another example from a commercial provider on why hard-breaking lines in shell examples is wrong and never done: https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-lamp-amazon-linux-2.html

Try resizing the page until the content gets a narrow column. You will see that the shell examples get a horizontal scroll bar, just like ours do. Azure docs do the same: https://docs.microsoft.com/en-us/azure/virtual-machines/linux/quick-create-cli

Reply:
I agree we shouldn't break command output, so if this exception is for this it's fine with me, especially as output is usually less useful and it's acceptable to not see it when looking at the page (ie. having horizontal scroll bar at the bottom).

Maybe we should include output only where it really makes sense as in example like https://docs.egi.eu/users/online-storage/object-storage/#access-via-s3-protocol it doesn't bring that much value.

We need to ensure that all files are styled in the same coherent way, prettier may have some choice that you may not like, but at least it provides an easy way for people to apply a common style, having plugins for various editors/IDE.

I can't agree with spaces at EOL or empty lines at EOF as I've got OCD with this, but if other agree with you we can disable the line length, there is an option for this in prettier: https://prettier.io/docs/en/options.html#prose-wrap.
I reckon that this quite simple change may make life easier for people editing files from the GitHub web interface.

Reply:
We are squashing PR so the details commits and last commits won't be seen. I'm/we are usually removing the minor commit messages from the final message, see https://github.com/EGI-Federation/documentation/commits/main.

If we don't have defined rules that are least enforced when validating the PR it will become a mess, no way we will disable this.

The idea was to have an automated formatter that can be used to ease people's life and help them with basic look and feel (useful for people less advanced than you on those topics) and then have automated checks when people are making commits so that we are sure to have a minimal quality for the submission. And of course both should be aligned. Using an "opinionated" formatter is also a way to end the discussions about style, as it is a sensitive topic :)

You are also free to propose a PR changing the formatter and linter settings (and the documentation/style guide).

Reply:
In fact you moved the enable line length check forward after a code excerpt that was not having any output, please break down this long command then let's try to finally merge this PR.